exporters/exporter_config.py

```diff
@@ -169,20 +180,36 @@ def _get_option_error(name, spec, config_options):
     return None
 
 
+def exported_classes(module):
+    names = module.__all__ if hasattr(module, '__all__') else dir(module)
+    classes = [(x, getattr(module, x)) for x in names]
+    return [(name, obj) for name, obj in classes if isclass(obj)]
+
+
 def _get_available_classes(module):
     classes_names = set()
-    for name, obj in getmembers(module):
-        if isclass(obj):
-            classes_names.add(obj.__module__ + '.' + obj.__name__)
+    for name, obj in exported_classes(module):
+        classes_names.add(obj.__module__ + '.' + obj.__name__)
     return classes_names
 
 
+def _get_module(module_name):
+    class_path_list = module_name.split('.')
+    mod = import_module('.'.join(class_path_list[0:-1]))
+    return getattr(mod, class_path_list[-1])
+
+
+def _is_stream_reader(config):
+    try:
+        reader = _get_module(config['reader']['name'])
+    except:
+        return True
```

Comment:
The get_module can fail for a lot of reasons (ImportError, KeyError, and anything else that can raise the imported module). we want to silence them because they will be checked by the code that checks the reader's config